webpack.config.web.js

```diff
@@ -24,6 +24,10 @@ const config = {
             __DEV__: false,
             'process.env.NODE_ENV': JSON.stringify('production')
         }),
+        /*
+         * This is used to pass environment variables to frontend when docker
+         * container initializes
+         */
```

Comment:
Sorry for being a pain, but this is not enough. We'll forget very soon. Please, could you at least add a link to the streambed issue?